src/dialects/postgres/query-generator-typescript.ts

```diff
@@ -48,4 +51,38 @@ export class PostgresQueryGeneratorTypeScript extends AbstractQueryGenerator {
       'GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;',
     ]);
   }
+
+  removeIndexQuery(tableName: TableNameOrModel, indexNameOrAttributes: string | string[], options: RemoveIndexQueryOptions) {
+    if (options) {
+      rejectInvalidOptions(
+        'removeIndexQuery',
+        this.dialect.name,
+        REMOVE_INDEX_QUERY_SUPPORTABLE_OPTIONS,
+        new Set(['concurrently', 'ifExists', 'cascade']),
+        options,
+      );
+
+      if (options.cascade && options.concurrently) {
+        throw new Error(`Cannot specify both concurrently and cascade options in removeIndexQuery for ${this.dialect.name} dialect`);
+      }
+    }
```

Comment:
```suggestion
    if (options?.cascade && options?.concurrently) {
      throw new Error(`Cannot specify both concurrently and cascade options in removeIndexQuery for ${this.dialect.name} dialect`);
    }
```
I think this should work as well. We normally do not use `rejectInvalidOptions` if all options are supported. TypeScript should help us so that we're not passing any other options

Reply:
Updated now